modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -361,6 +363,383 @@ Additionally, the user requires some `ReadOnly` permissions, and some of the rol
 
 For more information about creating an account with only the required privileges, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-5372F580-5C23-4E9C-8A4E-EF1B4DD9033E.html[vSphere Permissions and User Management Tasks] in the vSphere documentation.
 
+
+[discrete]
+[id="installation-vsphere-installer-infra--minimum-requirements_{context}"]
```

Comment:
Is this ID correct? The `--` in the middle seems unintentional.

Reply:
Fixed. I spotted this when changing the xref.